examples/large_models/Huggingface_pippy/model-config.yaml

```diff
@@ -0,0 +1,17 @@
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 100
+responseTimeout: 120
+parallelType: "pp"
+deviceType: "gpu"
+# deviceIds: [2,3] # seting CUDA_VISIBLE_DEVICES
```

Comment:
is the intent here to provide documentation as to avaialble configs? If so would prefer you explicitly mention the supported configs in the README directly